mutt/buffer.h

```diff
@@ -37,35 +37,48 @@ struct Buffer
-char *         mutt_buffer_strdup       (const struct Buffer *buf);
-void           mutt_buffer_seek         (struct Buffer *buf, size_t offset);
+void                mutt_buffer_alloc        (struct Buffer *buf, size_t size);
+void                mutt_buffer_dealloc      (struct Buffer *buf);
+void                mutt_buffer_fix_dptr     (struct Buffer *buf);
+struct Buffer *     mutt_buffer_init         (struct Buffer *buf);
+bool                mutt_buffer_is_empty     (const struct Buffer *buf);
+size_t              mutt_buffer_len          (const struct Buffer *buf);
+struct Buffer       mutt_buffer_make         (size_t size);
+void                mutt_buffer_reset        (struct Buffer *buf);
+char *              mutt_buffer_strdup       (const struct Buffer *buf);
+void                mutt_buffer_seek         (struct Buffer *buf, size_t offset);
 
 // Functions that APPEND to a Buffer
-size_t         mutt_buffer_addch        (struct Buffer *buf, char c);
-size_t         mutt_buffer_addstr       (struct Buffer *buf, const char *s);
-size_t         mutt_buffer_addstr_n     (struct Buffer *buf, const char *s, size_t len);
-int            mutt_buffer_add_printf   (struct Buffer *buf, const char *fmt, ...);
+size_t             mutt_buffer_addch        (struct Buffer *buf, char c);
+size_t             mutt_buffer_addstr       (struct Buffer *buf, const char *s);
+size_t             mutt_buffer_addstr_n     (struct Buffer *buf, const char *s, size_t len);
+int                mutt_buffer_add_printf   (struct Buffer *buf, const char *fmt, ...);
 
 // Functions that OVERWRITE a Buffer
-size_t         mutt_buffer_concat_path  (struct Buffer *buf, const char *dir, const char *fname);
-size_t         mutt_buffer_concatn_path (struct Buffer *dst, const char *dir, size_t dirlen, const char *fname, size_t fnamelen);
-size_t         mutt_buffer_copy         (struct Buffer *dst, const struct Buffer *src);
-int            mutt_buffer_printf       (struct Buffer *buf, const char *fmt, ...);
-size_t         mutt_buffer_strcpy       (struct Buffer *buf, const char *s);
-size_t         mutt_buffer_strcpy_n     (struct Buffer *buf, const char *s, size_t len);
-size_t         mutt_buffer_substrcpy    (struct Buffer *buf, const char *beg, const char *end);
+size_t             mutt_buffer_concat_path  (struct Buffer *buf, const char *dir, const char *fname);
+size_t             mutt_buffer_concatn_path (struct Buffer *dst, const char *dir, size_t dirlen, const char *fname, size_t fnamelen);
+size_t             mutt_buffer_copy         (struct Buffer *dst, const struct Buffer *src);
+int                mutt_buffer_printf       (struct Buffer *buf, const char *fmt, ...);
+size_t             mutt_buffer_strcpy       (struct Buffer *buf, const char *s);
+size_t             mutt_buffer_strcpy_n     (struct Buffer *buf, const char *s, size_t len);
+size_t             mutt_buffer_substrcpy    (struct Buffer *buf, const char *beg, const char *end);
```

Comment:
I think most of the reindentation in this file is superfluous.

Reply:
fixed